chore(linux): update libkeymancore.symbols after epic/core/9999-normalization merge

```
[linux/scripts] build.sh parameters: <--gha --bin-pkg /home/runner/work/keyman/keyman/artifacts/keyman-binarypkgs/libkeymancore_17.0.257-1~master-2[18](https://github.com/keymanapp/keyman/actions/runs/7718963934/job/21041507827#step:5:19)4.1+jammy1_amd64.deb --git-sha verify --git-base e4af[19](https://github.com/keymanapp/keyman/actions/runs/7718963934/job/21041507827#step:5:20)c8bab7b6463b33077f308ab91[21](https://github.com/keymanapp/keyman/actions/runs/7718963934/job/21041507827#step:5:22)776c8d9 verify>
[linux/scripts] ## verify starting...
[linux/scripts] OK: libkeymancore API didn't change
[linux/scripts] ERROR: libkeymancore.symbols file got changed without changing the package version number of the symbol
[linux/scripts] Major API change: 6 lines deleted and 1 lines modified
[linux/scripts] ERROR: Major API change without updating API version number in libkeymancore.symbols file
[linux/scripts] OK: API version in .symbols file and in CORE_API_VERSION.md is the same
[linux/scripts] ## verify failed
Error: Process completed with exit code 2.
```

https://github.com/keymanapp/keyman/actions/runs/7718963934/job/21041507827

--- a/linux/debian/libkeymancore.symbols
+++ b/linux/debian/libkeymancore.symbols
@@ -8,7 +8,7 @@ libkeymancore.so.1 libkeymancore #MINVER#
  km_core_context_items_dispose@Base 17.0.195
  km_core_context_length@Base 17.0.195
  km_core_context_set@Base 17.0.195
- km_core_cp_dispose@Base 17.0.244
+ km_core_cp_dispose@Base 17.0.263
  km_core_event@Base 17.0.195
  km_core_get_engine_attrs@Base 17.0.195
  km_core_keyboard_dispose@Base 17.0.195
@@ -22,11 +22,11 @@ libkeymancore.so.1 libkeymancore #MINVER#
  km_core_process_event@Base 17.0.195
  km_core_process_queued_actions@Base 17.0.195
  km_core_state_action_items@Base 17.0.195
- km_core_state_app_context@Base 17.0.245
+ km_core_state_app_context@Base 17.0.263
  km_core_state_clone@Base 17.0.195
  km_core_state_context@Base 17.0.195
  km_core_state_context_clear@Base 17.0.197
- km_core_state_context_debug@Base 17.0.244
+ km_core_state_context_debug@Base 17.0.263
  km_core_state_context_set_if_needed@Base 17.0.197
  km_core_state_create@Base 17.0.195
  km_core_state_debug_get@Base 17.0.195

